Source/WebCore/css/CSSCustomPropertyValue.cpp

```diff
@@ -79,6 +79,11 @@ String CSSCustomPropertyValue::customCSSText() const
             return serializeURL(value.string());
         }, [&](const String& value) {
             return value;
+        }, [&](const RefPtr<TransformOperation>& value) {
+            auto cssValue = transformOperationAsCSSValue(*value, RenderStyle::defaultStyle());
+            if (!cssValue)
+                return emptyString();
+            return cssValue->cssText();
```

Comment:
Line-saving opportunity!

```suggestion
            if (auto cssValue = transformOperationAsCSSValue(*value, RenderStyle::defaultStyle()))
                return cssValue->cssText();
            return emptyString();
```